### DISCUSSION TITLE
Installing `@ngneat/transloco-scoped-libs` results in dependency tree error

### DISCUSSION THREAD

**Discussion by valpioner** [ngneat/transloco#670] [Q&A]
Hello, we have a projects based on Angular 14/15 and when we try to install `@ngneat/transloco-scoped-libs` we get an error:

> npm i @ngneat/transloco-scoped-libs
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: ng15-test@0.0.0
npm ERR! Found: @angular/core@15.2.9
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"^15.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"13.3.10" from @ngneat/transloco-scoped-libs@3.0.5
npm ERR! node_modules/@ngneat/transloco-scoped-libs
npm ERR!   dev @ngneat/transloco-scoped-libs@"^3.0.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

I have checked transloco-scoped-libs [package.json](https://github.com/ngneat/transloco/blob/master/libs/transloco-scoped-libs/package.json) and there is no such dependency or peer-dependency on angular/core

1st question: 
So why does it try to install angular13 as a peer dependency?

2nd question
Is it safe to install it using `npm i --legacy-peer-deps` ?

**Comment by shaharkazaz** [comment]
I'll publish the fix this week, see #668

**Reply by valpioner** [reply]
Hi @shaharkazaz , may we expect new version (maybe `@ngneat/transloco-scoped-libs@3.0.6` ?) with this fix anytime soon?

**Reply by shaharkazaz** [reply]
@valpioner fixed in the latest version 👍
